packages/@aws-cdk/aws-servicecatalog/lib/private/association-manager.ts

```diff
@@ -98,7 +153,19 @@ export class AssociationManager {
     };
   }
 
+  private static stackSetConstraintLogicalId(associationKey: string): string {
+    return `StackSetConstraint${associationKey}`;
+  }
+
+  private static launchRoleConstraintLogicalId(associationKey:string): string {
+    return `LaunchRoleConstraint${associationKey}`;
+  }
+
   private static prettyPrintAssociation(portfolio: IPortfolio, product: IProduct): string {
     return `- Portfolio: ${portfolio.node.path} | Product: ${product.node.path}`;
   }
+
+  private static validateConstraintDescription(association: string, options: CommonConstraintOptions): void {
```

Comment:
I would just make this `validateCommonConstraintOptions`, or something. No reason to emphasize the description so much.